packages/wdio-jasmine-framework/src/reporter.js

```diff
@@ -32,7 +36,20 @@ export default class JasmineReporter {
         this.testStart = new Date()
         test.type = 'test'
         test.start = new Date()
-        this.parent[this.parent.length - 1].tests++
+        let parentSuite = this.parent[this.parent.length - 1]
+
+        /**
+         * if jasmine test has no root describe block, create one
+         */
+        if (!parentSuite) {
+            log.warn(
+                'No root suite was defined! This can cause reporters to malfunction. ' +
+                'Please always start a spec file with describe("...", () => { ... }).'
+            )
+        } else {
+            parentSuite.tests++
```

Comment:
I guess we should always call this one to keep existing behavior

Reply:
@mgrybyk the existing behavior is an error being thrown as mentioned in your issue:

> Cannot read property 'tests' of undefined